src/gatsby-node.js

```diff
@@ -3,9 +3,14 @@ import LoadablePlugin from '@loadable/webpack-plugin';
 import { statsFilename, statsPath } from './constants';
```

Comment:
Should you remove the `import { unlinkSync } from 'fs';` and the `onPostBuild` method in this file?

Because you remove the `fs` _readFileSync_ on `src/gatsby-ssr.js`

By the way, amazing PR proposal!

Reply:
Yep, you're right. I'd removed it on my local implementation already. I've pushed the change onto the branch/PR. Thanks!